src/datepicker/hebrew/ngb-calendar-hebrew.ts

```diff
@@ -64,7 +64,7 @@ export class NgbCalendarHebrew extends NgbCalendar {
     return day === 0 ? 7 : day;
   }
 
-  getWeekNumber(week: NgbDate[], firstDayOfWeek: number) {
+  getWeekNumber(week: ReadonlyArray<NgbDate>, firstDayOfWeek: number) {
```

Comment:
Could we use the `readonly NgbDate[]` syntax for all related `ngb-calendar-XXX.ts` files ?